modules/persistent-storage-local-metrics.adoc

```diff
@@ -19,8 +19,13 @@
 
 * `lso_lv_provisioned_PV_count`: total number of provisioned PVs for `LocalVolume`
 
-To use these metrics, be sure to:
+To use these metrics, be sure to enable them by doing one of the following:
 
-* Enable support for monitoring when installing the Local Storage Operator.
+* Select the checkbox for "Enable Operator recommended cluster monitoring on this Namespace" when installing the Local Storage Operator from OperatorHub in the web console.
```

Comment:
Change to: "When installing the Local Storage Operator from *OperatorHub* in the web console, select the *Enable Operator recommended cluster monitoring on this Namespace* checkbox.